src/display.hpp

```diff
@@ -583,9 +583,62 @@ class display : public filter_context, public video2::draw_layering
 	map_labels& labels();
 	const map_labels& labels() const;
 
+	/** Holds options for calls to function 'announce' (@ref announce). */
+	class announce_options {
+	private:
```

Comment:
Not really an issue, but we usually keep the private class members below the public interface.